### ISSUE TITLE
[MediaView] MediaView is not shown

### ISSUE THREAD

**Issue by Juwols** [Samsung/Tizen.CircularUI#174]
**Describe the bug**
MediaPlayer can play video but MediaView is not shown when
  - AutoPlay : true
  - UsesEmbeddingControls : false

in MediaPlayer.
At this time, Height is zero.
When UsesEmbeddingControls  is true, it works well.

**To Reproduce**
<c:MediaView x:Name="mediaView">
    <c:MediaPlayer x:Name="player"
                Source="https://archive.org/download/BigBuckBunny_328/BigBuckBunny_512kb.mp4"
                AutoPlay="True" UsesEmbeddingControls="False"/>
</c:MediaView>

**Expected behavior**
MediaView is shown.
![mediaview_normal](https://user-images.githubusercontent.com/1029299/54580994-f1223980-4a4d-11e9-8ace-1c3a7d33f188.png)

**Screenshots**
Nothing is shown.

**Environment (please complete the following information):**
 - Target Product Galaxy Watch, Gear S3, Galaxy Watch Active
 - Tizen Version 4.0.0.3
 - Tizen.CircularUI 1.1.0

**Additional context**

**Comment by rookiejava** [comment]
By default, the `HorizontalOptions` and `VerticalOptions` of `MediaView` are `LayoutOptions.Fill`. And, `LayoutOptions.Fill` generally overrides size requests specified using the `HeightRequest` and `WidthRequest` properties. So, in case of `UsesEmbeddingControls` == `false`, **you should specify the size (`HeightRequest` and `WidthRequest`) of `MediaView` or set `HorizontalOptions` and `VerticalOptions` to `LayoutOptions.FillAndExpand` in your application code**. Please refer to [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/layouts/layout-options) for more details about layout options.

In short, please modify your code as below:
```xml
<c:MediaView x:Name="mediaView" HorizontalOptions="FillAndExpand" VerticalOptions="FillAndExpand">
<c:MediaPlayer x:Name="player"
Source="https://archive.org/download/BigBuckBunny_328/BigBuckBunny_512kb.mp4"
AutoPlay="True" UsesEmbeddingControls="False"/>
</c:MediaView>
```
or
```xml
<c:MediaView x:Name="mediaView" WidthRequest="360" HeightRequest="360">
<c:MediaPlayer x:Name="player"
Source="https://archive.org/download/BigBuckBunny_328/BigBuckBunny_512kb.mp4"
AutoPlay="True" UsesEmbeddingControls="False"/>
</c:MediaView>
```
